Update Readme.md (#1)
DevExpress-Examples/devextreme-datagrid-batch-editing-select-multiple-cells: Readme.md

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T361032)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# DataGrid for DevExtreme - How to implement selecting multiple cells with a keyboard for batch editing 

> This example illustrates how to allow an end user to select more than one cell for editing by holding the CTRL key. When the user changes the editor text, the text will be entered to all selected cells.

Reviewer: ```suggestion
This example illustrates how to allow a user to select more than one cell for editing by holding the CTRL key. When the user changes the editor text, the text is entered in all the selected cells.
```
